mailpoet/lib/AdminPages/PageRenderer.php

```diff
@@ -114,6 +114,7 @@ public function displayPage($template, array $data = []) {
     $subscribersCacheCreatedAt = $this->transientCache->getOldestCreatedAt(TransientCache::SUBSCRIBERS_STATISTICS_COUNT_KEY) ?: Carbon::now();
 
     $defaults = [
+      'current_page' => sanitize_text_field(wp_unslash($_GET['page'] ?? '')),
```

Comment:
Would it be better to put these WP functions in our wrapper and then call them through that? I'm honestly not sure what the risk is (if any) to calling WP functions directly instead of using `$this->wp` like the rest of the file

Reply:
Throughout our code base, we do not use the WPFunctions for `sanitize_text_field` and `wp_unslash`. Both methods are not reflected in WPFunctions either. I think at the moment the code sniffer would react and warn about user input not being sanitized, if we would use WPFunctions.